src/fidesops/ops/api/v1/endpoints/consent_request_endpoints.py

```diff
@@ -133,8 +136,57 @@ def consent_request_verify(
     return _prepare_consent_preferences(db, provided_identity)
 
 
-@router.patch(
+@router.post(
     CONSENT_REQUEST_PREFERENCES,
```

Comment:
I suspect the URL is incorrect here, this requires a consent_id to be in the URL, `/consent-request/<consent_request_id>/preferences` but you don't actually look up anything by the consent request id you only seem to care about the identity.

I suspect `consent-request/preferences` is what you want here.  Currently someone could send in POST /consent-request/blah/preferences.